README.md

```diff
@@ -31,38 +33,64 @@ const client = new SgidClient({
 })
 ```
 
+### Generate PKCE pair
```

Comment:
wondering whether we should be more cognisant of users who may be looking at this with no prior knowledge of OIDC.

e.g. we can minimise the number of unfamiliar terms. this heading can be "Generate code challenge and verifier" so we avoid mentioning PKCE, and stick to just "code challenge" and "code verifier".

e.g. we can handhold users a little more. if I wasn't familiar with OIDC and I didn't know what this was, I would probably just skip it. and even if I didn't skip it, I wouldn't know that I need to store the code verifier. so maybe we can include a line below along the lines of "This is a required step for OIDC ([more info](https://oauth.net/2/pkce/)). You should store the `codeVerifier` in the user's session so you can retrieve it later."

Reply:
yeah that completely makes sense!

Reply:
i've added some descriptions about what they are and how to use it! feel free to comment if you feel it can be improved further